anatomy-of-a-social-network.md

```diff
@@ -0,0 +1,180 @@
+# Why talk about the anatomy of a social network?
+In the early days of networking each network was constructed
+as its own monolith. This lead to many networks that could not
+talk to each other. To enable networks to interoperate we needed
+an understanding of the anatomy of a network in order to design
+interoperability at the most appropriate level.
+ 
+Most students of networking are at some point introduced to the osi
+seven layer networking model
+ 
+ * layer 7. Application
+    *  The user space layer for application specific logic
+    *  https for the web
+ * layer 6. Presentation
+    *  A translation layer for smoothing differences between environments 
+       e.g. CR, LF, CR LF for line endings
+ * layer 5. Session
+    *  The layer that maintains state and endpoint identity
+ * layer 4. Transport
+    *  Put the packets into an end-to-end format, fix errors, 
+       direct to correct end application
+    *  tcp/udp for the internet
+ * layer 3. Network
+    *  dealing with the routing and delivery of Packets 
+       to the correct network interface
+    *  IPv4, IPv6 for the internet
+ * layer 2. data link
+    *  Deals with the Frames and media access control.
+       Ensures that the bits can be divided into packets
+       Prevents transmitters from interfering with each other.
+    *  Ethernet for most current networks. 
+       (Wifi, Copper Ethernet, Fiber Ethernet)
+ * layer 1. physical
+    *  The actual physical phenomena that can be interpreted as bits.
+       Radio waves for WiFi and Satellite.
+       Light for fiber optics point to point freespace microwave.
+       Electricity for CAT6 and Twisted pair.
+
+This enabled the internet and the web to be born.
+If we want to move from a number of incompatible social networks to a
+social internet we need to identify and separate the various layers of a
+social network so we can build and advocate protocol experimentation and
+standardization on each of the layers while maintaining the ability
+for each application to innovate on top of the common protocols.
+Standardise where we see value in standers and compete everywhere else.
+ 
+# Layers
+ 
+## Durability
```

Comment:
I think we should summarize the layers more briefly, and add examples below

Reply:
moved the summary from concussions up